snippets/multi-arch-schedule-taint.adoc

```diff
@@ -0,0 +1,32 @@
+:_mod-docs-content-type: SNIPPET
+
+Tainting each node for a specific architecture:: You can taint a node to avoid the node scheduling workloads that are incompatible with its architecture. When your cluster uses a `MachineSet` object, you can add parameters to the `.spec.template.spec.taints` field to avoid workloads being scheduled on nodes with non-supported architectures.
++
+Before you add a taint to a node, you must scale down the `MachineSet` object or remove existing available machines. For more information, see _Modifying a compute machine set_.
++
+.Example machine set with taint set
+--
+[source,yaml]
+----
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineSet
+metadata: # ...
+spec:
+  # ...
+  template:
+    # ...
+    spec:
+      # ...
+      taints:
+      - effect: NoSchedule
+        key: multi-arch.openshift.io/arch
```

Comment:
```suggestion
        key: multiarch.openshift.io/arch
```

[suggestion] we're already using multiarch with no dash in similar labels/keys.

Reply:
Thanks @aleskandro - do you know if that has changed at some point? Not sure why the existing OCP documentation would have `multi-arch` rather than `multiarch` otherwise and want to make sure that change makes it into the right versions.